<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRpmgyroWidget</class>
 <widget class="QWidget" name="QRpmgyroWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_46">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>391</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>罗盘校准</string>
   </property>
   <widget class="QLabel" name="label_82">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>地磁偏角</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PSIOFF">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>22</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_48">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>校准</string>
    </property>
    <widget class="QPushButton" name="pushButton_CALSTART">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始校准</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CALEND">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>校准完成</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CALASK">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>数据查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CALSTOP">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>校准中止</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X增益：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_KX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y增益：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_KY">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X偏差：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_BX">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y偏差：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_BY">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_PSIOFFASK">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PSIOFFSET">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_43">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>发动机配置</string>
   </property>
   <widget class="QRadioButton" name="radioButton_RPMTYPE2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>23</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>外置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_132">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>定速仪类型：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_RPMTYPE1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>23</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>内置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RPMTYPESET">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RPMTYPEASK">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_97">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>油门舵机</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_COLSERVORANGEASK">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_COLSERVORANGEMAX">
       <property name="text">
        <string> 0 </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_134">
       <property name="text">
        <string>舵机极性：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_COLDIR">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>正向</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/dir1.png</normaloff>:/images/dir1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_133">
       <property name="text">
        <string>限位A</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_284">
       <property name="text">
        <string>限位B</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pushButton_COLSERVORANGESET">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_COLSERVORANGEMIN">
       <property name="text">
        <string> 0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QSlider" name="horizontalSlider_COLSERVORANGEMAX">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QSlider" name="horizontalSlider_COLSERVORANGEMIN">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QSlider" name="horizontalSlider_engoffset">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>微调</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_engoffset">
       <property name="text">
        <string> 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>油门位置设置</string>
    </property>
    <widget class="QSlider" name="horizontalSlider_ENG">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-150</number>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_IDLINGSET">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置怠速点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FLAMEOUTSET">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置熄火点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SPEEDTEST">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>测试定速点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SPEEDSET">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置定速点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_HIGHSPEEDSET">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置最高点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FLAMEOUTTEST">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>测试熄火点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_IDLINGTEST">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>测试怠速点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_HIGHSPEEDTEST">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>测试最高点</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_COL">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>86</y>
       <width>34</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string> 0 </string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_114">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>336</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>旋翼转速</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_135">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>339</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>齿轮比</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RPM1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RPM2">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>360</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_136">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_137">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>==</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_138">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发动机转速/RPM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RPM">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RPMASK">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RPMSET">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_41">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>定速增益</string>
    </property>
    <widget class="QLabel" name="label_RPMI">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>40</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string> 0 </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CONSTANT_SPEEDASK">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CONSTANT_SPEEDSET">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_RPMI">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>23</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
